.github/workflows/tests.yml

```diff
@@ -72,6 +72,7 @@ jobs:
       - name: Scan for security issues
         run: |
           bandit -r src
+          safety check
```

Comment:
Q: This safety check is something different than `Snyk`, right?

Reply:
Yes, they both check their independent databases for vulnerabilities.